packages/app/cypress/e2e/tooltips.cy.ts

```diff
@@ -0,0 +1,34 @@
+const TooltipTriggerSelector = '.v-popper'
+const TooltipContentSelector = '.v-popper__popper--shown'
+
+describe('App: Tooltips', () => {
+  beforeEach(() => {
+    cy.scaffoldProject('cypress-in-cypress')
+    cy.openProject('cypress-in-cypress')
+    cy.startAppServer('e2e')
+    cy.visitApp()
+  })
+
+  it('should layer beneath modals', () => {
```

Comment:
Does this make more sense as a CT test? You could mount a modal and render a tooltip inside it to verify behavior.